metl-core/src/main/java/org/jumpmind/metl/core/runtime/flow/FlowRuntime.java

```diff
@@ -498,6 +498,7 @@ protected void sendNotifications(Notification.EventType eventType) {
                                 + notification.getNotifyType() + "'");
                         transport = mailSession.getTransport();
                         MimeMessage message = new MimeMessage(mailSession.getSession());
+                        message.setFrom(new InternetAddress(mailSession.getSession().getProperty("mail.from")));
```

Comment:
Aren't we already setting the mail.from property in the constructor of MailSession?  If you set the "from" in the global settings screen, then it should be picking it up already.

Reply:
As you see in an issue I've posted days ago (https://github.com/JumpMind/metl/issues/774), I already set the "from" in admin section, but with mail rejected by SendGrid.